Makefile

```diff
@@ -0,0 +1,14 @@
+GO_TEST_FLAGS ?= -race -count=1 -v -timeout=10m
```

Comment:
Do you want to use the Makefile in the build step? Personally, I am not a big fan of Makefiles, seems to be a very "old" technology ;) Building this Go package should be straight forward.

Reply:
I think they are convenient because `make` is installed on every computer and CI. And it's a consistent way to run something like `make test`. Not using it for building per se, but more for consistently running a few go commands.

But indifferent to them.